### PR TITLE
Added doc inline completion

### PR DESCRIPTION
**Pull request by kaihaozhao**
Today usually a small comment completion could take more than 2 secs, to test, stop typing for at least 2 secs or see logs.

### CODE REVIEW

File: src/InlineCompletionItemProvider.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            }]
+        };
+        return result;
+    }
+
+    private shouldSkip(document: vscode.TextDocument, position: vscode.Position): boolean {
+        if (!isComment(position) 
+        || !checkLineLength(position) 
+        || !isEndOfLine(document, position) 
+        // TODO remove. Today only support java doc completion.
```
Comment (codemakerai-dev) [inline src/InlineCompletionItemProvider.ts:69]: Why only documentation?

File: vscode.proposed.inlineCompletionsAdditions.d.ts.bak [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (codemakerai-dev) [inline vscode.proposed.inlineCompletionsAdditions.d.ts.bak:1]: Do we need this file?

File: src/InlineCompletionItemProvider.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        || langFromFileExtension(document.fileName) !== Language.java) {
```
Comment (codemakerai-dev) [inline src/InlineCompletionItemProvider.ts:70]: And why only Java?

File: src/InlineCompletionItemProvider.ts (hunk 1 above)
Comment (kaihaozhao) [reply src/InlineCompletionItemProvider.ts:69]: Will add code

File: vscode.proposed.inlineCompletionsAdditions.d.ts.bak (hunk 2 above)
Comment (kaihaozhao) [reply vscode.proposed.inlineCompletionsAdditions.d.ts.bak:1]: No, will delete it

File: src/InlineCompletionItemProvider.ts (hunk 3 above)
Comment (kaihaozhao) [reply src/InlineCompletionItemProvider.ts:70]: This line is already removed

**Pull request merged**